packages/algob/src/internal/deployer.ts

```diff
@@ -199,6 +209,9 @@ class DeployerBasicMode {
    * passing template parameters always during loading logic signature.
    * @param name ASC name
    * @returns loaded logic signature from artifacts/cache/<file_name>.teal.yaml
+   * Deprecated: this function will be removed in the next release. Use mkContractLsig to
```

Comment:
Shall we use `@deprecated` ?

Reply:
What's the difference?

Reply:
I remember it will show notification when we use this function. Is it also show when you use Deprecated?

Reply:
Thanks @vuvth , yes we should use it. https://stackoverflow.com/a/62991642/297094

Reply:
done